playbooks/roles/caasp4/templates/skuba-terraform.tfvars.j2

```diff
@@ -0,0 +1,60 @@
+# This file is managed by ansible and based on terraform.tfvars.ci.example from github.com/SUSE/skuba
+image_name = "{{ image_name }}"
+internal_net = "{{ internal_net }}"
+internal_subnet = "{{ internal_subnet }}"
+internal_router = "{{ internal_router }}"
+external_net = "{{ external_net }}"
+stack_name = "{{ stack_name }}"
+subnet_cidr = "{{ subnet_cidr }}"
+"dns_nameservers" = [
+    "8.8.8.8",
+    "8.8.8.4"
+]
+masters = 1
+workers = 2
```

Comment:
are 2 workers enough? We are using 3 right now. Might be worth using deploy_on_openstack_caasp_workers.

Reply:
2 is the number that the caasp team uses. Are you ok with leaving this for now to see how it goes?

Reply:
We currently use the first two caasp workers for the airship control plane, and the third caasp worker is the airship compute worker. Do we have a reason to ignore `deploy_on_openstack_caasp_workers`?

Reply:
ok. Fixed and it uses `deploy_on_openstack_caasp_workers` now